python/tvm/relay/backend/executor.py

```diff
@@ -27,7 +27,7 @@
 class Executor(Object):
     """Executor configuration"""
 
-    name = "executor"
+    registry_name = "executor"
 
     def __init__(self, name, options=None) -> None:
```

Comment:
can we clarify both name fields here? also suggest "flag_registry_name" or some other more narrowly-scoped use of registry, since there's tvm packedfunc registry as well

Reply:
renamed to flag_registry_name